packages/apps/human-app/server/src/modules/job-assignment/interfaces/job-assignment.interface.ts

```diff
@@ -0,0 +1,157 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { AutoMap } from '@automapper/classes';
+import {
+  SortField,
+  SortOrder,
+  StatusEnum,
+} from '../../../common/enums/job-assignment';
+
+export class JobAssignmentDto {
+  @AutoMap()
+  @ApiProperty({ example: 'string' })
+  exchange_oracle_url: string;
```

Comment:
Don't use the url here, use the address or can even get the address of the Exchange Oracle from the escrow contract.